### PR TITLE
6220 tele op

### PR DESCRIPTION
**Pull request by ThatSmartGuy21**
This pull request contains our BaseOpMode, our BaseTeleOp, and a test TeleOpCompetition. It allows for basic control of the robot and provides a basic framework for later programming endeavors.

### CODE REVIEW

**Comment by johnfraser98077** [comment]
Thanks for the invite, but I don’t review software.

File: Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+abstract public class BaseOpMode extends LinearOpMode {
```
Comment (satkab) [inline Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java:6]: I think it should be public first, then abstract

Comment (SJJCoding) [reply Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java:6]: There's no difference iirc

**Comment by johnfraser98077** [comment]
It would be very helpful if people would stop using nonsensical handles and used their real names to identify themselves in communications. I, for one, am at a loss to figure out who to talk to at the shop when confronted with “Not Debby Ryan,” for example. Not trying to single anyone out, there are a lot of these, including impersonating other students. 

Have pity on poor old fogeys!

Dr. Fraser

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (tmicro) [inline .idea/.gitignore:1]: Don't merge this file.  Please remove it.

**Review by tmicro — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 2 above)
Comment (ThatSmartGuy21) [reply .idea/.gitignore:1]: removed .gitignore

**Review by tmicro — APPROVED** [review]

**Review by sockeye-d — APPROVED** [review]

**Pull request merged**